apps/frontend/.env.example

```
@@ -1,2 +1 @@
-EXPO_PUBLIC_API_URL=https://universal-starter.onrender.com
-EXPO_PUBLIC_STORAGE_PREFIX=universal-starter
\ No newline at end of file
+EXPO_PUBLIC_API_URL=https://universal-starter.onrender.com
```

dotenv-linter (3.3.0)

[warning] [EndingBlankLine] No blank line at the end of the file

(EndingBlankLine)